### PR TITLE
Complete week 1 for eugenefedoto

### PR DESCRIPTION
**Pull request by eugenefedoto**
If everything is fine, you can usually comment something like:
:thumbsup:
or
lgtm (looks good to me)

I've also seen 🚀 . (type : for emoji)

#1 (don't forget to link the issue)

### CODE REVIEW

File: cs50/week-1/eugenefedoto/hello.c [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include "cs50.h"
+
+int main(void)
+{
+    string name = get_string("What's your name?\n");
+    printf("Hello, %s.\n", name);
```
Comment (eugenefedoto) [inline cs50/week-1/eugenefedoto/hello.c]: test

Comment (eugenefedoto) [inline cs50/week-1/eugenefedoto/hello.c]: sdgsdgs

File: cs50/week-1/eugenefedoto/hello.c [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include "cs50.h"
+
+int main(void)
+{
```
Comment (eugenefedoto) [inline cs50/week-1/eugenefedoto/hello.c]: test2

File: cs50/week-1/eugenefedoto/hello.c [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
```
Comment (eugenefedoto) [inline cs50/week-1/eugenefedoto/hello.c]: test3

File: cs50/week-1/eugenefedoto/hello.c (hunk 1 above)
Comment (eugenefedoto) [inline cs50/week-1/eugenefedoto/hello.c]: Multiple lines select

https://github.com/eugenefedoto/learning-c/blob/d32650e6a5bd5bec44e548977e79754de29615d7/cs50/week-1/eugenefedoto/hello.c#L5-L8

**Comment by tiazahmd** [comment]
👍

**Comment by eugenefedoto** [comment]
@tiazahmd You need to approve the PR, after you've checked it.

image

**Comment by eugenefedoto** [comment]
I just noticed I forgot to do the 3 other problems.

**Pull request merged**